### DISCUSSION TITLE
Problem with 360º image viewer

### DISCUSSION THREAD

**Discussion by bjhewitt** [thegetty/quire#895] [Get Help] [answered]
Hi, all. I'm having a problem with 360º images on catalog entry pages.

When clicking on 360º image on catalog entry page, the full screen viewer opens to first media item on page as part of a paginated set of all images instead of 360º image. So for example, if I have:

- a video as the first item in the entry content area (#1), 
- a 360º image second (#2) 
- and two static images following (#3, #4), 
- plus the main entry image (#5)

when I click on the 360º image, the full screen viewer open to item #1 (the video). This is only true of the 360º image - static images open correctly to a full screen view of themselves. I tried both having them all part of a figuregroup and as separate figures, which didn't seem to make any difference.

Codec/content/data available on GitHub at https://github.com/bjhewitt/quire-cmog-roman-glassblowing. It's a private repo (it's an unlaunched publication still), but happy to allow access to anyone who'd help me troubleshoot this!

a page demonstrating this issue is /catalogue/Compound-Objects/catobj--74/

Any help would be greatly appreciated!

**Comment by Erin-Cecele** [comment]
Hi @bjhewitt can you please give me access to your repo? I want to get in there and take a look. I also plan on testing this myself so if there are any other details you think might be relevant to my testing, please let me know!

**Comment by bjhewitt** [comment]
Hi, Erin. Thanks. You should have access now. Happy for any other feedback you may have. One thing I was curious about is whether there’s a way to add zoom controls to the IIIF zoom viewer.
________________________________

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @bjhewitt, fortunately, I have an easy, albeit fussy, answer. The issue you are encountering is related to #869. In order to get the lightbox feature working correctly, you will need to remove the periods from your IDs. I got the example below to work in your project:

```
  - id: "360-87-1-2"
    label: "360 image"
    caption: ""
    sequences:
      - id: "figures/360/87.1.2_360"
        behavior:
          - continuous
          - sequence
```
```
{% figuregroup 1, 'vid-52, 360-87-1-2' %}
```

At Getty, we typically only use "-" in our ids. The documentation was only recently updated to reflect this recent discovery of errors related to id formatting in v1. In the long term, we hope to fix this issue, but for now, you will need to follow the guidance reflected in the documentation: https://quire.getty.edu/docs-v1/figure-images/#create-a-figuresyaml-file-for-figure-image-metadata. Please confirm you are able to get this working on your end. 

Also, per your question about adding controls to the IIIF zoom viewer, this is on our roadmap as reflected in IIIF QA issue #815. However, we do not have a timeline for completion.

**Comment by bjhewitt** [comment]
Ah, thank you! I also missed the note in the documentation that figure IDs should always begin with a letter (not number - which makes sense from a JavaScript perspective), which I've also corrected now. Those changes have fixed this issue.
________________________________
